### PR TITLE
Added Python Solution for Cowntact Tracing (Bronze)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
all test cases pass for this solution.

### CODE REVIEW

**Review by dongliu0426 — APPROVED** [review]
clean code

**Review by brad182 — APPROVED** [review]
nice

**Comment by ryanchou-dev** [comment]
👍 ?

File: solutions/usaco-1037.mdx [hunk 1]
```diff
@@ -2,19 +2,98 @@
...
+
+n, t = map(int, input().split())
+
+infected = [int(i) for i in input().strip()]
+shakes = []
+
+for _ in range(t):
+	shakes.append(list(map(int, input().split())))
+
+# sorting a matrix automatically sorts lists the first element.
```
Comment (maggie-j-liu) [inline solutions/usaco-1037.mdx:33]: ? did you mean sorts by the first element or smth?

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/usaco-1037.mdx (hunk 1 above)
Comment (ryanchou-dev) [reply solutions/usaco-1037.mdx:33]: yeah

Comment (ryanchou-dev) [reply solutions/usaco-1037.mdx]: fixed it.

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**